samples/CommunityToolkit.Maui.Sample/CommunityToolkit.Maui.Sample.csproj

```diff
@@ -1,8 +1,8 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFrameworks>net6.0-ios;net6.0-android;net6.0-maccatalyst</TargetFrameworks>
-    <TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('windows')) and '$(MSBuildRuntimeType)' == 'Full'">$(TargetFrameworks);net6.0-windows10.0.19041</TargetFrameworks>
```

Comment:
@VladislavAntonyuk do you know why `and '$(MSBuildRuntimeType)' == 'Full'` was removed from this file?

Reply:
Because it was removed from MAUI template

Reply:
Thank you, I can see the reason why here which sounds like a problem I'm getting on Windows:
https://github.com/dotnet/maui/pull/5875
https://github.com/dotnet/maui/issues/5886